src/builders/enzymes.jl

```diff
@@ -60,7 +60,7 @@ function isozyme_flux_constraints(
         rid => C.Constraint(
             C.sum(
                 kcat(rid, iid) * i.value for (iid, i) in ri if !isnothing(kcat(rid, iid));
-                init = zero(C.LinearValue),
+                init = zero(C.LinearValue), # TODO not type stable if LinearValueT{notFloat64}
```

Comment:
I guess you'll have to capture the T from `kcat` and use it here?

on top:
```julia
    kcat::T,
) where T
```

here:
```julia
    init = zero(C.LinearValueT{T})
```

Reply:
...also with some sensible constraint on the T, like `<:Real` or so.